interface/wx/event.h

```diff
@@ -3644,7 +3644,127 @@ class wxScrollEvent : public wxCommandEvent
     void SetPosition(int pos);    
 };
 
+/** @class wxTouchEventBase
+    This is the base class for all touch events.
 
+    @note Touch events are not generated by default, you must call
+          wxWindow::EnableTouchEvents() with the appropriate parameter to
+          request their generation.
+
+    @library{wxcore}
+    @category{events}
+
+    @see wxGestureEvent, wxMultiTouchEvent
+
+    @since 3.1.1
+*/
+class wxTouchEventBase : public wxEvent
+{
+    /**
+        Constructor.
+    */
+    wxTouchEventBase(wxWindowID winid = 0, wxEventType type = wxEVT_NULL);
+
+    /**
+        Returns the position where the event took effect, in client coordinates: position of Pan event,
+        center of zoom for Zoom event, center of rotation for Rotate event, center of box formed by 2 fingers
+        for Two Finger Tap event and position of the pressed finger for Press and Tap Event.
+    */
+    const wxPoint& GetPosition() const;
+
+    /**
+        Sets the position where the event took effect, in client coordinates: position of Pan event,
+        center of zoom for Zoom event, center of rotation for Rotate event.
+    */
+    void SetPosition(const wxPoint& pos);
```

Comment:
I don't think this one really needs to be documented, it's for use by wxWidgets only, isn't it?